### PR TITLE
feat: set by property path

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: src/ParagraphsPastePluginBase.php [hunk 1]
```diff
@@ -50,8 +73,78 @@ public static function create(ContainerInterface $container, array $configuratio
...
+
+    // If bundle is defined, it's an ER field.
+    if ($bundle) {
+
+      $target_type = $fieldConfig->getFieldStorageDefinition()
+        ->getSetting('target_type');
+
+      /** @var \Drupal\Core\Entity\EntityInterface $newEntity */
+      $newEntity = $this->entityReferenceSelectionManager->getSelectionHandler($fieldConfig)
+        ->createNewEntity($target_type, $bundle, 'Created by paragraphs paste', 0);
```
Comment (ol0lll) [inline src/ParagraphsPastePluginBase.php]: Label will be same for every entity? Why bother setting it?
Why is uid set to 0?

**Review by ol0lll — CHANGES_REQUESTED** [review]

File: src/ParagraphsPastePluginBase.php (hunk 1 above)
Comment (chrfritsch) [reply src/ParagraphsPastePluginBase.php]: Looks like we can pass NULL as a name and for the User we should get the current user

Comment (ol0lll) [reply src/ParagraphsPastePluginBase.php]: 👍

File: src/ParagraphsPastePluginBase.php [hunk 2]
```diff
@@ -50,8 +73,78 @@ public static function create(ContainerInterface $container, array $configuratio
...
+
+      $entity->{$fieldName}[] = $newEntity;
+      if (!empty($property_path)) {
+        $this->setValue($property_path, $newEntity, $value);
+      }
+    }
+    else {
+      $entity->{$fieldName} = $value;
+    }
+    $entity->save();
```
Comment (ol0lll) [inline src/ParagraphsPastePluginBase.php]: Saving the entity at this point will lead to stale entities in database with no parent (host) entity in paragraphs case. Pass it through to the build function to only have it in the form.

**Review by ol0lll — CHANGES_REQUESTED** [review]

**Review by ol0lll — APPROVED** [review]

**Pull request merged**